packages/keycloak-user-management/src/components/forms/UserForm/index.tsx

```diff
@@ -69,6 +69,29 @@ const UserForm: FC<UserFormProps> = (props: UserFormProps) => {
     form.setFieldsValue(initialValues);
   }, [form, initialValues]);
```

Comment:
i think we don't need this anymore , do we ?

Reply:
It breaks tests. Form props don't update without it